### DISCUSSION TITLE
Installation advice on Mac M1

### DISCUSSION THREAD

**Discussion by PhilP88** [lukas-blecher/LaTeX-OCR#100] [Q&A] [answered]
Hi,
I successfully installed this amazing project on windows, but I can't get it working on Mac. Has anyone succeeded? 
I tried to install all packages via Rosetta but was still getting error messages, had something to do with open_cv as far as I remember. 
Any help would be appreciated.
Thanks

**Comment by PhilP88** [comment, marked as answer]
I might want to give an update. You have to install everything via rosetta, including python. Even then I only manage to get pix2tex work.

**Comment by PhilP88** [comment]
I might want to give another update. It's working! Big thanks to the creators.

**Reply by lukas-blecher** [reply]
Thanks for the update
